cmd/bnscli/cmd_sigs.go

```diff
@@ -0,0 +1,114 @@
+package main
+
+import (
+	"encoding/hex"
+	"encoding/json"
+	"errors"
+	"flag"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/iov-one/weave/cmd/bnsd/app"
+	"github.com/iov-one/weave/cmd/bnsd/client"
+	"github.com/iov-one/weave/crypto"
+)
+
+func cmdSignTransaction(
+	input io.Reader,
+	output io.Writer,
+	args []string,
+) error {
+	fl := flag.NewFlagSet("", flag.ExitOnError)
+	fl.Usage = func() {
+		fmt.Fprint(flag.CommandLine.Output(), `
+Sign given transaction. This is decoding a transaction data from standard
+input, adds a signature and writes back to standard output signed transaction
+content.
+
+`)
+		fl.PrintDefaults()
+	}
+	var (
+		tmAddrFl = fl.String("tm", "https://bns.NETWORK.iov.one:443", "Tendermint node address. Use proper NETWORK name.")
```

Comment:
It is kind of nice to allow defining `-tm` via environmental variable (or config file), as this would be used by all invocations. The script wrappers you proposed could have something like `export TM_ADDR=${TM_ADDR:-https://...}.` on the top and then not worry about the flags and be very easy to customize.

Reply:
This environmental variable technique is used here:
https://github.com/iov-one/weave/blob/master/cmd/bnsd/main.go#L22-L23

Reply:
Done